package.json

```diff
@@ -7,7 +7,9 @@
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "flow": "flow"
+    "flow": "flow",
+    "lint": "eslint .",
+    "fix": "eslint . --fix --ext .js"
```

Comment:
I find this redundant. I'd say `yarn lint --fix` is enough. Or at least the script to be `lint:fix`. Just `fix` is very generic.

Reply:
Yeah I would leave the script for fixing but as @ferrannp said `fix` is too generic. Let's use `lint:fix`